tests/unit_node/_fs/_fs_handle_test.ts

```diff
@@ -317,3 +317,100 @@ Deno.test({
     await fileHandle.close();
   },
 });
+
+Deno.test(
+  "[node/fs filehandle.createReadStream] Create a read stream",
+  async function () {
+    const fileHandle = await fs.open(testData);
+    const stream = fileHandle.createReadStream();
+    const fileSize = (await fileHandle.stat()).size;
+
+    assertEquals(stream.bytesRead, 0);
+    assertEquals(stream.readable, true);
+
+    let bytesRead = 0;
+
+    stream.on("open", () => assertEquals(stream.bytesRead, 0));
+
+    stream.on("data", (data) => {
+      assertEquals(data instanceof Buffer, true);
+      assertEquals((data as Buffer).byteOffset % 8, 0);
+      bytesRead += data.length;
+      assertEquals(stream.bytesRead, bytesRead);
+    });
+
+    stream.on("end", () => {
+      assertEquals(stream.bytesRead, fileSize);
+      assertEquals(bytesRead, fileSize);
+    });
+
+    // Wait for the 'close' event so that the test won't finish prematurely
+    await new Promise<void>((resolve) => {
+      stream.on("close", resolve);
+    });
+    
+    // await fileHandle.close();
+  }
+);
+
+Deno.test(
+  "[node/fs filehandle.createWriteStream] Create a write stream",
```

Comment:
I'm not sure if these tests are meant to be taken from somewhere

The definition of the `createReadStream` test I took from #27591 
The definition of the `createWriteStream` I wrote myself trying to follow the same style